src/test/java/test/ConfirmationCodeKeTest.java

```diff
@@ -0,0 +1,45 @@
+package test;
+
+import client.ApiClientException;
+import org.testng.annotations.Test;
+import pages.ConfirmationCodePage;
+import providers.KeConfirmationCodeStringProviders;
+
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertTrue;
+
+public class ConfirmationCodeKeTest extends TestBase {
+
+    KeConfirmationCodeStringProviders keConfirmationCodeStringProviders = new KeConfirmationCodeStringProviders();
+
+    @Test
+    public void enterConfirmationCode() throws IOException, ApiClientException {
```

Comment:
Bad test name. it is not clear what are you testing in this test

Reply:
Fixed. Renamed method to `testEnterConfirmationCodeToSetPin`